app/ide-desktop/lib/dashboard/src/hooks/eventCallback.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * @file useEventCallback shim from the latest React RFC.
+ */
+
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
avoid whole-file lint disables - especially without explanation

Reply:
For this file it makes sense, bc without any we can't infer the types properly

Reply:
yes, but even then `any` is only used in two places. is there a reason why we can't disable the lint in those two places individually?